docs/source/storage/general_storage.rst

```diff
@@ -41,27 +39,7 @@ You can schedule an online consultation on the `RC website <https://rc.northeast
    If you are not connected to the campus internet, you must be connected to
    the university's VPN (GlobalProtect) before you can access these storage systems.
    You can find detailed information about downloading and using the GlobalProtect VPN
-   in the `ServiceNow Knowledge Base <https://service.northeastern.edu/tech?id=kb_article&sys_id=4701e07adb93485084ba5595ce9619a9>`_.
-
-Connecting to /research
-========================
-After you have received notification that your shared storage folder on /research is ready for you to use, you
-can connect to it on your laptop. Use the following procedures to connect to /research from either a Windows PC or a Macbook.
-If you are not on the campus internet, you will need to first be connected to Northeastern's VPN before you can connect to /research.
-
-**Windows**
+   in the `ServiceNow Knowledge Base <https://service.northeastern.edu/tech?id=kb_article_view&sysparm_article=KB0013983>`_.
```

Comment:
I just updated this to the permalink. 

Now that I'm looking again, maybe we should update to change from "ServiceNow Knowledge Base" to "FAQ: VPN and remote access." Just a thought.